source/documentation/get-started.md

```diff
@@ -61,7 +61,7 @@ You can also accept your invitation by signing in to GitHub and visiting the [or
 
 When you sign in to the Analytical Platform for the first time, you will be prompted to configure additional two-factor authentication (2FA) for the Analytical Platform itself.
 
-To sign in, go to the Analytical Platform [control panel](https://controlpanel.services.alpha.mojanalytics.xyz/)or the [control panel for EKS-based analytical platform](https://controlpanel.services.analytical-platform.service.justice.gov.uk/).
```

Comment:
can you leave this in please as mentioned above - lots of people are still on the old platform and if they click the new link they'll be automatically upgraded and will lose any local files they hadn't backed up